bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/component.tsx

```diff
@@ -167,6 +175,31 @@ const ChatMessageList: React.FC<ChatListProps> = ({
     }
   };
 
+  const renderUnreadNotification = () => {
```

Comment:
Please, use [useMemo](https://react.dev/reference/react/useMemo#usememo) to avoid uncessary recomputations between renders